antora.yml

```diff
@@ -7,7 +7,7 @@ nav:
 asciidoc:
   attributes:
     neo4j-version: '5'
-    neo4j-version-minor: '5.25'
-    neo4j-version-exact: '5.25.0'
-    neo4j-buildnumber: '5.25'
+    neo4j-version-minor: '5.26-LTS'
```

Comment:
Do we need to use 'LTS' here? I think the title 'Neo4j 5.26 LTS' will be in the version-picker.
I just don't remember where we use this attrib `neo4j-version-minor`.